content/hi/docs/reference/glossary/pod-disruption.md

```diff
@@ -0,0 +1,22 @@
+---
+title: पोड डिसृपशन (Pod Disruption)
+id: pod-disruption
+date: 2021-05-12
+full_link: /docs/concepts/workloads/pods/disruptions/
+short_description: >
+  एक प्रक्रिया जिससे नोड्स के ऊपर जो पॉड्स है वो अपनी इच्छा से या फिर उसके विपरीत समाप्त हो सके। 
+  
+aka:
+related:
+ - pod
+ - container
+ tags:
+  - operation
```

Comment:
The checks on this PR are failing. if you check netlify logs you'll find: ```9:30:40 AM: Error: Error building site: "/opt/build/repo/content/hi/docs/reference/glossary/pod-disruption.md:11:1": failed to unmarshal YAML: yaml: line 11: did not find expected '-' indicator```

I checked the file in `en` and spotted the indentation of tags: and - operation are incorrect. Please fix these before moving forward.

Reply:
Done the changes.